dropwizard-index-page/src/main/java/com/palantir/indexpage/Asset.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.indexpage;
+
+import com.github.mustachejava.Mustache;
+import javax.annotation.CheckForNull;
+
+/**
+ * Simple asset interface used by both {@link ClasspathAsset} and {@link FileSystemAsset} to return the compiled asset
+ * template.
+ */
+public interface Asset {
+
+    @CheckForNull
+    Mustache getTemplate();
```

Comment:
this is confusing. why does an asset return a `Mustache`. wouldn't it just return content?